applications/rag/notebook_starter.py

```diff
@@ -0,0 +1,17 @@
+import requests
+import sys
+
+api_url = 'http://127.0.0.1:9443/hub/api'
+
+token = sys.argv[1].strip()
+print(token)
+
+r = requests.post(api_url + '/users/admin/server',
+    headers = {
+        'Authorization' : f'token {token}',
+    },
+    json={},
+)
+
+if r.raise_for_status():
```

Comment:
The if and raise exception below is not needed, I think you can just run `r.raise_for_status()` since it raises an exception. For example: https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/applications/rag/tests/test_frontend.py#L6